backend/doc/migration.md

```diff
@@ -34,7 +34,397 @@ and activate the Tracim virtualenv:
 
 ### Creating a New Schema Migration ###
 
-This creates a new auto-generated python migration file 
+This creates a new auto-generated python migration file
 in `tracim_backend/migration/versions/` ending with `migration_label.py`:
 
     alembic -c development.ini revision --autogenerate -m "migration label"
+
+
+# Creating working migration #
+
+Creating a working alembic migration for all supported database of tracim is not so easy,
+here are some trick to help us.
+
+## sqlite support of alter table is limited, use always batch_op !
+
+sqlite database has limited support for alter table ops, which mean you can't easily add/remove/rename columns.
```

Comment:
SQLite has

ops altering tables,

which mean you can't easily  -> As a consequence, you can't […]